deployment/ansible/deploy.yml

```diff
@@ -167,7 +167,7 @@
 
       - name: Update epoch configuration
         template:
-          src: templates/epoch.yaml.j2
+          src: templates/epoch.yaml
```

Comment:
Yes, from the docs: http://jinja.pocoo.org/docs/dev/templates/

> A Jinja template doesn’t need to have a specific extension: .html, .xml, or any other extension is just fine.

I didn't tried another extension because @jsnewby didn't able to provide a list of extensions that would confuse GH pages. I'm too lazy to play guess the extension game with GH pages :)
Moreover, I think the directory name that contains the file is pretty self explanatory.